apps/web/public/static/locales/en/common.json

```diff
@@ -2855,6 +2855,8 @@
   "routing_form_insights_booking_status_header": "Booking",
   "routing_form_insights_response_no_booking": "No Booking",
   "routing_form_insights_booked_by": "Booked by",
+  "routing_form_insights_booking_uid": "UID",
+  "routing_form_insights_booking_link": "Link",
```

Comment:
these probably dont need such a unique key and can likely be reused later, no? @eunjae-lee

Reply:
i would just do

"uid": "UID",
"link": "Link"

Reply:
good point. made the change.